pkg/server/home_workspaces.go

```diff
@@ -0,0 +1,661 @@
+/*
+Copyright 2021 The KCP Authors.
```

Comment:
22

Reply:
Fixed in commit a1e36e31d467daafd64d6cf57f61268794951ea3